src/main/java/com/xiao/users/dto/UserDto.java

```diff
@@ -19,6 +22,8 @@ public class UserDto {
     @Size(min = 8, message = "Password length must be greater then 8")
     private String password;
 
-    @Email(message = "Email is not valid")
+    @Email(message = "Email is not valid", groups = EmailValidation.class)
```

Comment:
why do we still have the groups = EmailValidation.class here?
Does it serve any purpose?